apps/performance-tests/src/Datasets.ts

```diff
@@ -4,8 +4,54 @@
  *--------------------------------------------------------------------------------------------*/
 import fs from "fs";
 import path from "path";
+import { insertPhysicalElement, insertPhysicalModelWithPartition, insertSpatialCategory } from "presentation-test-utilities/lib/IModelUtils";
+import { createIModel } from "./util/IModelUtilities";
 
-async function downloadDataset(name: string, downloadUrl: string, localPath: string): Promise<void> {
+const LARGE_FLAT_IMODEL_SIZE = 50_000;
+const BAYTOWN_DOWNLOAD_URL = "https://github.com/imodeljs/desktop-starter/raw/master/assets/Baytown.bim";
+
+export class Datasets {
+  private static readonly _iModels: { baytown?: string; largeFlat?: string } = {};
+
+  public static get bayTown(): string {
+    return this.verifyInitialized(this._iModels.baytown);
+  }
+
+  public static get largeFlat(): string {
```

Comment:
I'd rename "large flat" (wherever it's currently used) to "50k elements" to make it clear what to expect.

Reply:
Renamed the module to "50k elements" and refactored for easier addition of flat iModels in the future